network/gossip/libp2p/test/epochtransition_test.go

```diff
@@ -102,14 +91,17 @@ func (ts *EpochTransitionTestSuite) TearDownTest() {
 	}
 }
 
-// TestNewNodeAdded tests that an additional node in the next epoch gets connected to other nodes and can exchange messages
-// in the current epoch
-func (ts *EpochTransitionTestSuite) TestNewNodeAdded() {
+// TestNewNodeAdded tests that when a new node is added to the identity list
+// (ie. as a result of a EpochSetup event) that it can connect to the network.
+func (ts *MutableIdentityTableSuite) TestNewNodeAdded() {
+
 	// create the id, middleware and network for a new node
 	ids, mws, nets := generateIDsMiddlewaresNetworks(ts.T(), 1, ts.logger, 100, nil, false)
+	newID := ids[0]
```

Comment:
```suggestion
	newID := ids[0]
	ts.nets = append(ts.nets, nets[0])
```

Reply:
didn't realize this earlier, but the new node won't be shutdown in the teardown if we don't include its network (`net[0]`) in `ts.nets`